### PR TITLE
Adds blog post on workflow CLI

### PR DESCRIPTION
**Pull request by bmbouter**

### CODE REVIEW

File: _posts/2018-09-14-pulp3-cli-plan.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+examples of HTTPie usage, .i.e. `http` in the
+[pulp_file instructions](https://github.com/pulp/pulp_file#create-a-repository-foo). Using HTTPie is
+similar to the auto-generating CLI because both have each command be a single API call.
+
+
+# Recommendation to Pulp3 users
+
+So for the time being, at least through the Pulp 3.0 General Availability, it's recommended to use
+[HTTPie](https://httpie.org/) for your commend line needs. You can see examples of its usage in the
+[pulp_file instructions](https://github.com/pulp/pulp_file#create-a-repository-foo).
```
Comment (daviddavis) [inline _posts/2018-09-14-pulp3-cli-plan.md]: Add: "Other Pulp3 plugins have similar instructions on how to interact with their APIs using httpie in their README files"

File: _posts/2018-09-14-pulp3-cli-plan.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+
+# Workflow Based CLI
+
+The Pulp2 CLI, `pulp-admin` is primarily a workflow tool. Pulp3 does several steps differently than
+Pulp2, so as users approach Pulp3, we don't yet know the workflow use cases that are of broad use to
+many users. Given that requirements gap, the current approach is to wait to hear from users who are
+building workflows for Pulp3 so we can help may their lives easier through a workflow-based CLI.
```
Comment (asmacdo) [inline _posts/2018-09-14-pulp3-cli-plan.md]: s/may/make/

File: _posts/2018-09-14-pulp3-cli-plan.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Another option that is available is to use [HTTPie](https://httpie.org/) as the CLI. You can see
+examples of HTTPie usage, .i.e. `http` in the
+[pulp_file instructions](https://github.com/pulp/pulp_file#create-a-repository-foo). Using HTTPie is
+similar to the auto-generating CLI because both have each command be a single API call.
+
+
+# Recommendation to Pulp3 users
+
+So for the time being, at least through the Pulp 3.0 General Availability, it's recommended to use
+[HTTPie](https://httpie.org/) for your commend line needs. You can see examples of its usage in the
```
Comment (dkliban) [inline _posts/2018-09-14-pulp3-cli-plan.md]: s/commend/command/

File: _posts/2018-09-14-pulp3-cli-plan.md (hunk 1 above)
Comment (bmbouter) [reply _posts/2018-09-14-pulp3-cli-plan.md]: added in next push

File: _posts/2018-09-14-pulp3-cli-plan.md (hunk 2 above)
Comment (bmbouter) [reply _posts/2018-09-14-pulp3-cli-plan.md]: ty, fixed in next push

File: _posts/2018-09-14-pulp3-cli-plan.md (hunk 3 above)
Comment (bmbouter) [reply _posts/2018-09-14-pulp3-cli-plan.md]: ty, fixed in next push

**Pull request merged**